### PR TITLE
Fix me: broken update to dev dependencies

### PR DESCRIPTION
**Pull request by bovine3dom**
I get the following errors. I think they are all from updating TypeScript. It seems to have decided that `HTMLElement.focus()` needs an argument amongst other things.

```
ℹ ｢atl｣: Using typescript@3.0.3 from typescript
ℹ ｢atl｣: Using tsconfig.json from /home/olie/projects/tridactyl/tsconfig.json
ℹ ｢atl｣: Checking started in a separate process...
Error: /home/olie/projects/tridactyl/src/.excmds_content.generated.ts(1497)
 Property 'fullscreenElement' does not exist on type 'Document'.
Error: /home/olie/projects/tridactyl/src/.excmds_content.generated.ts(1643)
 Expected 1 arguments, but got 0.
Error: /home/olie/projects/tridactyl/src/commandline_frame.ts(85)
 Expected 1 arguments, but got 0.
Error: /home/olie/projects/tridactyl/src/commandline_frame.ts(88)
 Expected 1 arguments, but got 0.
Error: /home/olie/projects/tridactyl/src/content/commandline_content.ts(84)
 Expected 1 arguments, but got 0.
Error: /home/olie/projects/tridactyl/src/excmds.ts(1929)
 Property 'fullscreenElement' does not exist on type 'Document'.
Error: /home/olie/projects/tridactyl/src/excmds.ts(2102)
 Expected 1 arguments, but got 0.
Error: /home/olie/projects/tridactyl/src/lib/dom.ts(469)
 Expected 1 arguments, but got 0.
ℹ ｢atl｣: Time: 1099ms
Warning: Unknown language: "text"
Warning: Unknown language: "text"
Rendering [========================================] 100%
```

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -135,7 +135,7 @@
     },
     "@types/events": {
       "version": "1.2.0",
-      "resolved": "https://registry.npmjs.org/@types/events/-/events-1.2.0.tgz",
+      "resolved": "http://registry.npmjs.org/@types/events/-/events-1.2.0.tgz",
```
Comment (bovine3dom) [inline package-lock.json]: Now with a more concise URL. That's progress for you!

File: src/tridactyl.d.ts [hunk 2]
```diff
@@ -16,6 +16,11 @@ interface Window {
     eval(str: string): any
 }
 
+// Firefox Document interface
```
Comment (antonva) [inline src/tridactyl.d.ts]: We should probably strive to flesh this out and maybe upstream it to web-ext-types?

File: src/completions.ts [hunk 3]
```diff
@@ -212,7 +212,7 @@ export abstract class CompletionSourceFuse extends CompletionSource {
         return [undefined, undefined]
     }
 
-    fuseOptions = {
+    fuseOptions: Fuse.FuseOptions<any> = {
```
Comment (antonva) [inline src/completions.ts]: This should probably be actually typed but it works.

**Comment by antonva** [comment]
Idk why travis is throwing a hissy fit. But it builds locally.

**Comment by glacambre** [comment]
I think you might have to remove your node_modules directory and re-run `npm install` in order to reproduce Travis's issue.

**Comment by bovine3dom** [comment]
Works fine for me. I think the issue is with the Travis build process. I can't see where that's configured - .travis-ci.yml doesn't really seem to say anything. It might still be on @cmcaine's Travis account?

**Comment by bovine3dom** [comment]
Squished and merged.

**Comment by antonva** [comment]
Gosh dangit!

**Pull request closed without merging**